cmake/ucx-config.cmake.in

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (C) NVIDIA Corporation. 2021.  ALL RIGHTS RESERVED.
+#
+
+set(UCX_LIBRARIES "@prefix@/lib")
```

Comment:
`@libdir@` (it could be lib64)

Reply:
`@libdir@` will be replaced by `${exec_prefix}/lib`, but in the context of cmake, `exec_prefix` is not defined. But I can do something like
```cmake
set(prefix "@prefix@")
set(exec_prefix "@exec_prefix@")

set_target_properties(ucx::ucs PROPERTIES
  IMPORTED_LOCATION "@libdir@/libucs.so"
  INTERFACE_INCLUDE_DIRECTORIES "@includedir@"
)
```